### DISCUSSION TITLE
Is it compulsory to use postgresSql with medusa. Can i use neon with medusa.

### DISCUSSION THREAD

**Discussion by Fardeen-Awais** [medusajs/medusa#4742] [Q&A] [answered]
I apologize if it is a silly question but i'm new on medusa. I want to know about how can i use it with neon or any other database service like supabase.

**Comment by ramzlab000** [comment]
Or there must be a way to specify the credentials after installation!

**Comment by SGFGOV** [comment]
Yes you can use neon

Best Regards
Govind Diwakar

On Fri, 11 Aug, 2023, 16:17 ramzlab000, ***@***.***> wrote:

> Or there must be a way to specify the credentials after installation!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/medusajs/medusa/discussions/4742#discussioncomment-6700362>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHGJS7ZUUGY6YFOOGKDXUYEVHANCNFSM6AAAAAA3LSVONY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.

**Reply by Fardeen-Awais** [reply]
Can you elaborate how?

**Comment by SGFGOV** [comment, marked as answer]
You just need to use the string neon provides

Best Regards
Govind Diwakar

On Fri, 11 Aug, 2023, 23:48 Fardeen Awais, ***@***.***> wrote:

> Can you elaborate how?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/medusajs/medusa/discussions/4742#discussioncomment-6704303>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHD5S262YIK6VDB7W3TXUZZPXANCNFSM6AAAAAA3LSVONY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.
